SubmittingPatches-backports.rst

```diff
@@ -0,0 +1,344 @@
+======================================
+Submitting Patches to Ceph - backports
+======================================
+
+Most likely you're reading this because you intend to submit a GitHub pull
+request ("PR") targeting one of the stable branches ("nautilus", etc.) at
+https://github.com/ceph/ceph.
+
+Before you open that PR, please read (at the very least) the following general
+rules::
+
+Backports should state (either in the backport PR or in the tracker issue):
+
+1. what bug is fixed
+2. why this fix is the minimal way to do it
+3. why does this need to be fixed in <release>
```

Comment:
i agree that we should always take these into consideration when backporting a fix. but i am not sure we should/could enforce these.

Reply:
These are guidelines that @athanatos suggested long ago. Note that I wrote "backports should state", not "backports *must* state". I am not suggesting that we start enforcing this in the sense of "rejecting backports that don't include explicit statements satisfying these guidelines", but I think that these guidelines are a good starting point when considering whether, and how, to backport something.

Reply:
but, i think unlike guidelines, "rules" are supposed to be followed.